skymp5-server/cpp/addon/ScampServer.cpp

```diff
@@ -272,10 +272,10 @@ ScampServer::ScampServer(const Napi::CallbackInfo& info)
     partOne->worldState.AttachScriptStorage(scriptStorage);
 
     partOne->AttachEspm(espm);
-    this->serverSettings = serverSettings;
+    this->serverSettings = std::move(serverSettings);
```

Comment:
Actually, I'd prefer keeping this code without `std::move`. it'd be simpler: no one will use invalidated `serverSettings` instead of `this->serverSettings` by mistake.

parsing settings isn't a bottle-neck. we do it once. and settings size is const so it's O(1). let's not `std::move` here

Reply:
Okay.